packages/plugins/other/visitor-plugin-common/src/base-resolvers-visitor.ts

```diff
@@ -1021,6 +1054,21 @@ export class BaseResolversVisitor<
     };
   }
 
+  private getContextType(parentName: string, node: FieldDefinitionNode): string {
+    let contextType = this._fieldContextTypeMap[`${parentName}.${node.name}`]
+      ? this._fieldContextTypeMap[`${parentName}.${node.name}`].type
+      : 'ContextType';
+
+    for (const directive of node.directives) {
```

Comment:
If a given node matches both a `fieldContextTypes` and multiple `directiveContextExtender`,
Isn't it supposed to merge the context types together?
Also, we need a test to cover this scenario 👀 

**Example**

```gql
# schema.graphql
type Mutation {
  notAuthMutaiton(): [NotAuthRes!]!
  loggedMutation (content: String!): loggedRes! @isLoggedIn
}

type Query {
  loggedQuery: LoggedRes!  @isLoggedIn
}
```

```yaml
# codegen.yml
generates:
  ./src/graphql/resolvers/types.generated.ts:
    plugins:
      - typescript
      - typescript-resolvers
    config:
      contextType: ./context.ts#Context
    fieldContextTypes:
         - Query.loggedQuery#./my-file#ExternalContextType
     directiveContextExtender:
      - isLoggedIn: ./context.ts#LoggedContext
```

Reply:
I've add tests to check this!